insights/parsers/lsscsi.py

```diff
@@ -0,0 +1,91 @@
+"""
+LsSCSI - command ``/usr/bin/lsscsi``
+====================================
+
+This parser reads the output of ``/usr/bin/lsscsi`` into a list of dictionarys.
+Each item is a dictionary with six keys:
+
+* ``HCTL`` - the scsi_host,channel,target_number,LUN tuple
+* ``Peripheral-Type`` - the SCSI peripheral type
+* ``Vendor`` - the vendor name
+* ``Model`` - the model name
+* ``Revision`` - the revision string
+* ``Primary-Device-Node`` - the primary device node name
+
+This dictionary is available in the ``data`` attribute.
```

Comment:
Normally attributes are defined using the `Attributes:` keyword as recommended in the documentation guidelines.  You might consider changing this to be clearer about the structure of the `data` attribute.  For example:
```
Attributes:
    data (list of dict): List of the input lines, where each line is a dictionary having the keys identified above.
```